aimmo-game/tests/test_simulation/maps.py

```diff
@@ -23,11 +23,10 @@ def serialise(self):
 
 
 class MockCell(Cell):
-    def __init__(self, location=1, habitable=True, generates_score=False,
-                 avatar=None, pickup=None, name=None, actions=[]):
+    def __init__(self, location=1, habitable=True, avatar=None,
+                 pickup=None, name=None, actions=[]):
```

Comment:
I think you changed `actions` here(the bug PR from yesterday), maybe you should merge/rebase?